experiment_params/default_environments/three_bodies.yaml

```diff
@@ -0,0 +1,5 @@
+environment:
+  name: "NObjectGravity"
+  mass: [0.5, 0.5, 0.5]
```

Comment:
```suggestion
  mass: [1., 1., 1.]
```

Reply:
Thanks, I was not sure about the default parameters. Are all the others okay?